spec/ruby/optional/capi/thread_spec.rb

```diff
@@ -84,5 +84,10 @@ def call_capi_rb_thread_wakeup
 
       lambda { thr.join }.should raise_error(NotImplementedError)
     end
+
+    it "sets the thread's group" do
+      thread_group = @t.rb_thread_create(lambda {}, nil).group
+      thread_group.class.should == ThreadGroup
```

Comment:
Being picky here, but you can use `.....group.should be_an_instance_of(ThreadGroup)` if I'm not mistaken.